etc/config/c++.amazon.properties

```diff
@@ -196,10 +198,10 @@ compiler.clang800.semver=8.0.0
 compiler.clang800.options=--gcc-toolchain=/opt/compiler-explorer/gcc-8.3.0
 compiler.clang900.exe=/opt/compiler-explorer/clang-9.0.0/bin/clang++
 compiler.clang900.semver=9.0.0
-compiler.clang900.options=--gcc-toolchain=/opt/compiler-explorer/gcc-9.2.0
+compiler.clang900.options=--gcc-toolchain=/opt/compiler-explorer/gcc-9.3.0
```

Comment:
I don't think we want to change the gcc version used on old stable clangs - notice clang800 still uses `gcc-8.3.0`.
This comment also applies to every other stable compiler that gets a `--gcc-toolchain=` option.